###########################################################################
# This patch was contributed by Marius Strobl.
###########################################################################
# The contents of this file are subject to the terms of the
# Common Development and Distribution License, Version 1.0 only
# (the "License").  You may not use this file except in compliance
# with the License.
#
# See the file CDDL.Schily.txt in this distribution for details.
#
# When distributing Covered Code, include this CDDL HEADER in each
# file and include the License file CDDL.Schily.txt from this distribution.
###########################################################################
# The file CDDL.Schily.txt can be found in the original cdrtools tarball,
# which is mirrored at ftp://ftp.freebsd.org/pub/FreeBSD/distfiles/.
###########################################################################

--- scgcheck/scgcheck.1.orig	Mon Sep 29 14:56:16 2003
+++ scgcheck/scgcheck.1	Mon Sep 29 14:56:44 2003
@@ -234,8 +234,7 @@
 .SH SEE ALSO
 .BR cdrecord (1),
 .BR readcd (1),
-.BR mkisofs (1),
-.BR scg (7).
+.BR mkisofs (8).
 
 .SH NOTES
 .PP
